default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib
+, stdenv
+, hyprland
+,
+}:
+stdenv.mkDerivation {
+  pname = "hycov";
+  version = "0.1";
+  src = ./.;
+
+  inherit (hyprland) nativeBuildInputs;
+
+  buildInputs = [ hyprland ] ++ hyprland.buildInputs;
+}
```

Comment:
might be nice to add the meta attributes

mainly description, platforms, license and maintainer(s)

Reply:
Done